cookbook/templating/namespaced_paths.rst

```diff
@@ -74,6 +74,10 @@ directory:
             );
         ));
 
+.. caution::
+
+    These paths are not warmed during cache warming. This is fixed in 2.8.
```

Comment:
Our doc managers may have further comments for you, but personally I love your rewording because I can fully understand it now. Thanks!